.github/workflows/ci.yaml

```diff
@@ -92,36 +84,30 @@ jobs:
           - repository: quay.io/tinkerbell/tink-controller
             binary: tink-controller
     steps:
-      - name: Docker Image Tag for Sha
+      - name: Create docker image tags
         id: docker-image-tag
-        run: |
-          echo ::set-output name=tags::${{ matrix.repository }}:latest,${{ matrix.repository }}:sha-${GITHUB_SHA::8}
-      - name: Checkout code
-        uses: actions/checkout@v2
+        run: echo ::set-output name=tags::${{ matrix.repository }}:latest,${{ matrix.repository }}:sha-${GITHUB_SHA::8}
+      - uses: actions/checkout@v3
       - name: Login to quay.io
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         if: ${{ startsWith(github.ref, 'refs/heads/main') }}
         with:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
```

Comment:
dont we need qemu for Docker multi-arch builds?

Reply:
Nope. I think this is a relic of when multi-arch builds included binary compilation. The system now cross compiles the binaries in an earlier job. `docker build` doesn't need QEMU to build arch specific images.